tools/plugins/com.liferay.ide.server.tomcat.core/src/com/liferay/ide/server/tomcat/core/PortalTomcatBundle.java

```diff
@@ -261,6 +263,15 @@ protected int getDefaultJMXRemotePort() {
 
 		args.add("-Dcatalina.base=\"" + bundlePath.toPortableString() + "\"");
 		args.add("-Dcatalina.home=\"" + bundlePath.toPortableString() + "\"");
+
+		Version portalVersion = new Version(getVersion());
+
+		if (portalVersion.compareTo(new Version("7.0.0")) > 0) {
```

Comment:
hey @simonjhy  where do these properties come from?

Reply:
Can you create a new ticket and describe what problem this is solving and then send a separate PR ?  I'm not going to merge these changes in this PR.  Please go ahead and send a followup PR for this.

Reply:
this properties get from portal-impl.jar.the method getVersion is existed method of portalBundle class

Reply:
these three new arguments are come from tomcat 9.0.1 original parameters.

Reply:
Got it.  Thanks for checking that.  Please those in a new update.

Reply:
ok